data/web/debug.php

```diff
@@ -43,6 +43,11 @@
         <?php
           $exec_fields = array('cmd' => 'system', 'task' => 'df', 'dir' => '/var/vmail');
           $vmail_df = explode(',', json_decode(docker('post', 'dovecot-mailcow', 'exec', $exec_fields), true));
+          $used_percent = substr($vmail_df[4], 0, -1);
+          $quota_stats = mailbox('get','quota_stats');
+          $quotaPercent1 = round(($quota_stats['total_quota']/$vmail_df[3])*100);
+          $quotaPercent2 = round((($quota_stats['total_quota']-$quota_stats['used_bytes'])/$vmail_df[3])*100);
+          $quotaPercent2 = ($quotaPercent2+$used_percent>100) ? 100-$used_percent : $quotaPercent2;
```

Comment:
Handling overcommitment -> progressbar is capped to 100%

Reply:
Mörciii